docs/common-tasks/opening-urls-in-vms.md

```diff
@@ -98,7 +98,7 @@ Disposable VMs are based on the following templates:
 - dvm-online (minimal with firefox only)
 - dvm-anon (whonix workstation)
 
-AppVMs are highly specialized: vault (offline), documents (offline), media (offline), dev (firewalled), email (firewalled). Those is where information lives. But files do not get opened nor worked on there ... only on instances of dvm-offline.
+AppVMs are highly specialized: vault (offline), documents (offline), media (offline), email (firewalled). Those is where information lives. But files do not get opened nor worked on there ... only on instances of dvm-offline.
```

Comment:
dev VM is a bad example as one typically works with text files that are 
- well known
- text only
- created by oneself or trusted source
so it might not be practical to edit each source code file in a DispVM especially when relying on a modern IDE